pkg/blockfactory/blockissuer.go

```diff
@@ -43,14 +43,19 @@ type BlockIssuer struct {
 	optsTipSelectionTimeout       time.Duration
 	optsTipSelectionRetryInterval time.Duration
 	optsIncompleteBlockAccepted   bool
+	optsRateSetterEnabled         bool
 }
 
 func New(p *protocol.Protocol, account Account, opts ...options.Option[BlockIssuer]) *BlockIssuer {
 	return options.Apply(&BlockIssuer{
-		events:     NewEvents(),
-		workerPool: p.Workers.CreatePool("BlockIssuer"),
-		protocol:   p,
-		Account:    account,
+		events:                      NewEvents(),
+		workerPool:                  p.Workers.CreatePool("BlockIssuer"),
+		protocol:                    p,
+		Account:                     account,
+		optsIncompleteBlockAccepted: false,
```

Comment:
What is `optsIncompleteBlockAccepted` supposed to indicate?

Reply:
It defines whether the node allows to fill in incomplete block and issue it for user.
 
 It's one of the node parameters. I'll add a comment in the block issuer as well to be clear.